app/services/preserved_object_handler.rb

```diff
@@ -277,124 +232,66 @@ def update_pc_validation_timestamps(pres_copy)
 
   # expects @incoming_version to be numeric
   def version_comparison_results(db_object, version_symbol)
-    results = []
     if incoming_version == db_object.send(version_symbol)
-      results << result_hash(VERSION_MATCHES, db_object.class.name)
+      handler_results.add_result(PreservedObjectHandlerResults::VERSION_MATCHES, db_object.class.name)
     elsif incoming_version < db_object.send(version_symbol)
-      results << result_hash(ARG_VERSION_LESS_THAN_DB_OBJECT, db_object.class.name)
+      handler_results.add_result(PreservedObjectHandlerResults::ARG_VERSION_LESS_THAN_DB_OBJECT, db_object.class.name)
     elsif incoming_version > db_object.send(version_symbol)
-      results << result_hash(ARG_VERSION_GREATER_THAN_DB_OBJECT, db_object.class.name)
+      handler_results.add_result(PreservedObjectHandlerResults::ARG_VERSION_LESS_THAN_DB_OBJECT, db_object.class.name)
```

Comment:
was changing the result code here intentional?  seems like this should stay as it was?  but also this didn't seem to break the tests, so i guess either i'm missing something or this is a test case we missed?

Reply:
Actually, it's a test case I fudged, so I'm delighted you caught it.

https://github.com/sul-dlss/preservation_catalog/blob/7120f7411f70ef97f9d479373475d614340a85d5/spec/services/preserved_object_handler_update_version_spec.rb#L169-L179